scripts/google_cloud_storage.py

```diff
@@ -0,0 +1,43 @@
+import os
+import json
+
+from google.cloud import storage
+
+
+class GoogleCloudStorage:
+    def __init__(self, bucket_name):
+        self.bucket_name = bucket_name
+        self.client = storage.Client()
+        self.bucket = self.client.get_bucket(self.bucket_name)
+
+    def upload_files(
+        self, output_files, directory, output_filename_prefix=None, **kwargs
+    ):
+        for output_file in output_files:
+            output_filename = (
+                f'{output_filename_prefix}_{output_file}'
+                if output_filename_prefix
+                else output_file
+            )
+
+            blob = self.bucket.blob(f'{directory or ""}/{output_filename}')
+            blob.metadata = {**kwargs}
+            blob.upload_from_filename(filename=output_file)
+
+    def get_files(self, output_dir):
+
+        for blob in self.client.list_blobs(self.bucket_name):
+
+            file_path = '/'.join(iter(blob.name.split('/')[:2]))
```

Comment:
Built-in `rsplit` could be used.
```suggestion
            file_path = blob.name.rsplit('/', 1)[0]
```